expand/expand_test.go

```diff
@@ -61,3 +63,40 @@ func TestConfigNils(t *testing.T) {
 		})
 	}
 }
+
+func TestFields(t *testing.T) {
```

Comment:
this isn't a generic table-driven test of Fields, so I'd prefer it if you called it something more precise like `TestFieldsIdempotency` or `TestFieldsRepeatedCalls`.